### PR TITLE
chore: remove unused types, exports, variables, functions, etc

### PR DESCRIPTION
**Pull request by tyleroooo**
Detected with a npm library I found called knip. I'm happy to add some back if people think they're useful, just wanted to default to removing dead code.

### CODE REVIEW

**Review by yogurtandjam — COMMENTED** [review]
imo could be useful to keep extra exports available for abacus. i like the idea of being able to rely on this file as a general source for all abacus typing needs.

if anything maybe we could move all the types to a `types/abacus.ts` file and knip ignore that one? open to ideas here though

**Comment by tyleroooo** [comment]
Happy to add them back but my impression is that it's already not close to exhaustive so might as well just keep it as like a list of shortcuts that we actually use.

File: src/components/ComingSoon.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useStringGetter } from '@/hooks/useStringGetter';
 
 import { layoutMixins } from '@/styles/layoutMixins';
 
-export const ComingSoon = () => {
```
Comment (jaredvu) [inline src/components/ComingSoon.tsx:9]: if we remove `export` here, we may as well just remove the component as a whole.

**Review by jaredvu — APPROVED** [review]

File: src/components/ComingSoon.tsx (hunk 1 above)
Comment (tyleroooo) [reply src/components/ComingSoon.tsx:9]: it's used below

**Pull request merged**